src/components/Navbar.js

```diff
@@ -31,8 +37,33 @@ export default function Navbar() {
               className={cx(styles.inputField, styles.widthFixed)}
             />
           </li>
+
+          <li>
+          <Checkbox
+                label="Snap to grid"
+                className={cx(styles.checkGrid)}
+              />
+          </li>
+          <li>
+          <Checkbox
+                label="Ratio"
+                className={cx(styles.checkRatio)}
+              />
+          </li>
+          <li>
+          <TextField
+              name="canvasColor"
+              hintText="#ffffff"
+              className={cx(styles.inputField, styles.widthFixed)}
+            />
+        </li>
+          <li>
+           <RaisedButton className={cx('btnEnterGrid', styles.btnEnterGrid)} label="Enter grid" secondary={true} />
```

Comment:
The thing here is that the first argument into the `cx` function species that the selector is global and the later that it is local. You should write it like this instead: `className={styles.btnEnterGrid}`